packages/amplify_api/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 0.4.0 (2022-02-17)
+## 0.4.1 (2022-02-28)
+
+- chore: clean up some code to limit analysis infos
```

Comment:
nit: would it be possible to keep the PR #s like past entries?

Reply:
Sounds good added those in